cacm_questions.py

```diff
@@ -1,17 +1,29 @@
 import numpy as np
 import matplotlib.pyplot as plt
+import os
 
 from math import log
 
 from models.document import CACMDocumentCollection
 
 
 if __name__ == '__main__':
-    document_collection = CACMDocumentCollection(
-        data_filename='Data/CACM/cacm.all',
-        stop_list_filename='Data/CACM/common_words',
-        load_on_creation=True,
-    )
+
+    if 'cacm.collection' in os.listdir('Data/Collection'):
+        print("ONE")
+        document_collection = CACMDocumentCollection(
+            data_filename='Data/CACM/cacm.all',
+            stop_list_filename='Data/CACM/common_words',
+            load_on_creation=False,
+        )
+        document_collection.load_from_file('Data/Collection/cacm.collection')
+    else:
+        print("TWO")
+        document_collection = CACMDocumentCollection(
+            data_filename='Data/CACM/cacm.all',
+            stop_list_filename='Data/CACM/common_words',
+            load_on_creation=True,
+        )
```

Comment:
You keep print here but you don't keep it in stanford_question, is it normal?

Reply:
No it's not, I forgot to remove it !